LamestWebserver/LamestWebserver/HttpResponse.cs

```diff
@@ -67,9 +67,17 @@ public class HttpResponse
         /// <summary>
         /// Creates a new HttpResponse
         /// </summary>
-        public HttpResponse()
+        public HttpResponse(HttpRequest req)
         {
             Date = DateTime.Now.ToString(HtmlDateFormat);
+            if(req != null)
+            {
+                Range = req.Range;
+                if(Range != null)
+                Status = "206 Partial Content";
```

Comment:
the problem is just a missing tab:
`if(Range != null)
    Status = "206 Partial Content";`